packages/design-system/src/components/N8nNotice/Notice.vue

```diff
@@ -102,7 +102,7 @@ export default Vue.extend({
 	font-size: var(--font-size-2xs);
 	display: flex;
 	color: var(--custom-font-black);
-	margin: var(--spacing-s) 0;
+	margin: var(--notice-margin, var(--spacing-s) 0);
```

Comment:
Nice! We should have variables with fallback values for all our components. Can you add a tech deb task for it?

Reply:
Of course! https://linear.app/n8n/issue/N8N-4465